src/wallet/interfaces.cpp

```diff
@@ -403,7 +404,24 @@ class WalletImpl : public Wallet
     CAmount getBalance() override { return GetBalance(*m_wallet).m_mine_trusted; }
     CAmount getAvailableBalance(const CCoinControl& coin_control) override
     {
-        return GetAvailableBalance(*m_wallet, &coin_control);
+        LOCK(m_wallet->cs_wallet);
+        CAmount total_amount = 0;
+        // Fetch selected coins total amount
+        if (coin_control.HasSelected()) {
+            FastRandomContext rng{};
+            CoinSelectionParams params(rng);
+            // Note: for now, swallow any error.
```

Comment:
Was remembering why I left it unhandled:
And it was because this cannot happen. `FetchSelectedInputs` fails only if any of the selected coins are not solvable or invalid. And we set the selected coins via the coin control dialog, which is loaded from the wallet's available UTXOs, which are always valid.

Reply:
I see. In that case, just add what you just wrote as a comment.

Reply:
Ok, sounds good. Will do it in the next push.